lib/core/elastic_connector_actions.rb

```diff
@@ -144,7 +144,6 @@ def fetch_document_ids(index_name)
         result
       end
 
-      # should only be used in CLI
```

Comment:
I think it's good to leave some comment here about why we don't use it but still keep it, maybe mentioning that index is normally created by Kibana or Enterprise Search.

Reply:
I remove this comment because it's no longer true.

Reply:
It's not used anywhere, right?

Reply:
It's used here:
https://github.com/elastic/connectors-ruby/blob/aac87fd740f52e965480af5936167f33589640ad/lib/app/dispatcher.rb#L94

And in CLI, which I think we need to re-visit later.